README.md

```diff
@@ -9,15 +9,76 @@ Here is an example of a Grafana dashboard for an ESP project. This dashboard rel
+
+### Connect a Panel to SAS Event Stream Processing as a Data Source
+1. Create a new dashboard and add a panel.
+2. In the **Query** tab at the bottom of the panel editor, select the data source that you configured previously.</br>The plug-in discovers running instances of ESP servers by connecting to your chosen data source. When the connection is successful, the **Query** tab shows drop-down menus that are related to SAS Event Stream Processing.
+3. Use the **ESP server** drop-down menu to select the ESP server that you want to query. You can filter the available options by entering a keyword and then selecting the desired ESP server in the drop-down menu.
+4. Use the **ESP project**, **Continuous query**, and **Window** drop-down menus to select appropriate values until you are able to narrow the query down to the desired target window in the ESP project.</br>When an available target window is selected, the plug-in establishes a connection and starts querying for new events.
+5. In the **Fields** drop-down menu, select the fields (from the window in your ESP project) that you want to visualize.
+6. In the top right corner of the screen, if required, change the visualization type from the default of **Time series** to a visualization type that suits your ESP project.
+
+> **Note**: 
+> - You can reuse existing queries across multiple panels, by selecting **--Dashboard--** as a data source and targeting the panel that contains the existing query.
+> - The dashboard that you create references the name of the ESP project. If you rename the ESP project or rename any windows in the ESP project, the dashboard no longer works. As a result, if you want to use the same dashboard with more than one ESP project, you must create a separate dashboard for each project.
+
+### Examples
+
+Some SAS Event Stream Processing Studio examples include Grafana dashboards:
+
+1. In SAS Event Stream Processing Studio, click ![Help](img/icon-helpmenu.png "Help") on any page and select **Examples**.
+
+2. Install the Sailing example or the ONNX example.
+
+3. Run the example in test mode.
+
+4. Download the Grafana dashboard to your computer:
+
+   - [Dashboard for the Sailing example](https://github.com/sassoftware/esp-studio-examples/tree/main/Advanced/sailing#visualizing-objects-in-grafana)
+
+   - [Dashboard for the ONNX example](https://github.com/sassoftware/esp-studio-examples/tree/main/Advanced/onnx_object_detection#visualizing-objects-in-grafana)
+
+5. Import the dashboard to Grafana.
+   
+## Contributing
+
+> We are not currently accepting contributions. 
+
+## License
+
+> This project is licensed under the [Apache 2.0 License](LICENSE).
+
+## Additional Resources
+- [Grafana documentation](https://grafana.com/docs/)
+- [Grafana tutorials](https://grafana.com/tutorials/)
+
+## SAS Internal Deployment Notes
+This section is only relevant to user working for SAS.
```

Comment:
How about:
"This section is relevant only to internal users at SAS."

Reply:
Done